tests/test_equilibrium.py

```diff
@@ -464,3 +466,156 @@ def test_backward_compatible_load_and_resolve():
     f_obj = ForceBalance(eq=eq)
     obj = ObjectiveFunction(f_obj)
     eq.solve(maxiter=1, objective=obj)
+
+
+def test_shifted_circle_geometry():
+    """
+    In this test, we calculate a low-beta shifted circle equilibrium with DESC.
+
+    We then compare the various geometric coefficients with their respective analytical
+    expressions.
+    """
+    R0 = 1
+    a0 = 0.1
+
+    surf = FourierRZToroidalSurface(
+        R_lmn=[R0, a0],
+        Z_lmn=[0.0, -a0],
+        modes_R=np.array([[0, 0], [1, 0]]),
+        modes_Z=np.array([[0, 0], [-1, 0]]),
+        sym=True,
+        NFP=1.0,
+    )
+
+    pressure = PowerSeriesProfile([1.5e5, 0, -1.5e5])
+    iota = PowerSeriesProfile([0.5, 0, -0.2])
+
+    eq = Equilibrium(
+        L=10, M=10, N=0, surface=surf, Psi=0.1, pressure=pressure, iota=iota
+    )
+
+    eq, _ = eq.solve(objective="force", ftol=1e-3, gtol=1e-6, maxiter=30, verbose=3)
```

Comment:
do we need to actually do the solve as part of this test? Or can we just load a saved version stored in `tests/inputs`?

Reply:
No, we don't really have to calculate it every time a test is run. I can add this equilibrium to tests/inputs.